src/test/java/com/scottlogic/filter/FilterContentKeywordTest.java

```diff
@@ -2,42 +2,86 @@
 import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
 import java.util.List;
 
 public class FilterContentKeywordTest {
 
-    FilterContentKeyword filterContentKeyword = new FilterContentKeyword();
+    @Nested
+    @DisplayName("Functionality with")
+    class filterContentKeyword_functionality {
+        @Test
+        @DisplayName("keyword: 'you'")
+        void setFilterContentKeyword_you() {
+            FilterContentKeyword filterContentKeyword = new FilterContentKeyword("you");
+            List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
+            List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_you();
+            Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter));
+        }
 
-    @Test
-    @DisplayName("Functionality with 'you'")
-    void setFilterContentKeyword_you() {
-        List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
-        List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_you();
-        Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter, "you"));
-    }
+        @Test
+        @DisplayName("keyword: 'You'")
+        void setFilterContentKeyword_You() {
+            FilterContentKeyword filterContentKeyword = new FilterContentKeyword("You");
+            List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
+            List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_You();
+            Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter));
+        }
 
-    @Test
-    @DisplayName("Functionality with 'You'")
-    void setFilterContentKeyword_You() {
-        List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
-        List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_You();
-        Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter, "You"));
+        @Test
+        @DisplayName("keyword: 'yuo'")
+        void setFilterContentKeyword_yuo() {
+            FilterContentKeyword filterContentKeyword = new FilterContentKeyword("yuo");
+            List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
+            List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_yuo();
+            Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter));
+        }
+
+        @Test
+        @DisplayName("keyword: 'Funky funky")
+        void setFilterContentKeyword_Funky_funky() {
+            FilterContentKeyword filterContentKeyword = new FilterContentKeyword("Funky funky");
+            List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
+            List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_Funky_funky();
```

Comment:
When expected values are simple (like an empty list or a small array) it is common for it to be set explicitly in the test.
It does add a few lines but it makes it more readable.

Usually we load json files into some DTO (a data model class) for integration tests because response types can be very long.

Just a suggestion for what to expect, I'll allow this implementation.